### ISSUE TITLE
Recent update results in error

### ISSUE THREAD

**Issue by nayab9** [layday/instawow#41]
recently ran an instawow update and now see the following:

```
Traceback (most recent call last):
  File "/usr/local/bin/instawow", line 8, in <module>
    sys.exit(main())
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 1257, in invoke
    sub_ctx = cmd.make_context(cmd_name, args, parent=ctx)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 700, in make_context
    self.parse_args(ctx, args)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 1048, in parse_args
    value, args = param.handle_parse_result(ctx, opts, args)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 1630, in handle_parse_result
    value = invoke_param_callback(self.callback, ctx, self, value)
  File "/home/nayab9/.local/lib/python3.8/site-packages/click/core.py", line 123, in invoke_param_callback
    return callback(ctx, param, value)
  File "/home/nayab9/.local/lib/python3.8/site-packages/instawow/cli.py", line 148, in <lambda>
    return lambda c, _, v: fn(c.obj.m, v)
  File "/home/nayab9/.local/lib/python3.8/site-packages/instawow/utils.py", line 108, in __get__
    o.__dict__[self.f.__name__] = v = self.f(o)
  File "/home/nayab9/.local/lib/python3.8/site-packages/instawow/cli.py", line 115, in m
    config = Config.read(self.ctx.params['profile']).ensure_dirs()
  File "/home/nayab9/.local/lib/python3.8/site-packages/instawow/config.py", line 94, in read
    dummy_config.migrate_legacy_dirs()
  File "/home/nayab9/.local/lib/python3.8/site-packages/instawow/config.py", line 137, in migrate_legacy_dirs
    legacy_config = self.parse_raw(legacy_config_file.read_text(encoding='utf-8'))
  File "pydantic/main.py", line 478, in pydantic.main.BaseModel.parse_raw
  File "pydantic/main.py", line 455, in pydantic.main.BaseModel.parse_obj
  File "pydantic/env_settings.py", line 33, in pydantic.env_settings.BaseSettings.__init__
  File "pydantic/main.py", line 346, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for _GlobalConfig
profile
  none is not an allowed value (type=type_error.none.not_allowed)
```
Instawow is in my $PATH; not sure what is causing this

**Comment by layday** [comment]
What command did you try to run?

**Comment by layday** [comment]
It looks like you might have a configuration where the value of `profile` for some reason is null.  What is the output of `cat /home/nayab9/.config/instawow/config.json`?

**Comment by beachcamp** [comment]
I updated and had the same problem. I resolved it by backuping up/deleting my config.json and running setup configuration again "instawow".

This generated a "/profiles/_default_" folder that did not previously exist. With contents identical to the root "/instawow/" folder.

I then copied my db.sqlite from the root "/instawow/db.sqlite" into "/profiles/_default_/db.sqlite" and re-ran "instawow update"

This time instead of having the error instawow was able to find all of my addons and update them as normal.
